tests/bdd/test_volume_topology.py

```diff
@@ -0,0 +1,536 @@
+"""Volume Topology feature feature tests."""
+
+import docker
+import pytest
+import requests
+from openapi_client.model.volume_policy import VolumePolicy
+from pytest_bdd import (
+    given,
+    scenario,
+    then,
+    when,
+)
+
+import common
+from openapi.openapi_client.model.create_pool_body import CreatePoolBody
+from openapi.openapi_client.model.create_volume_body import CreateVolumeBody
+from openapi.openapi_client.model.protocol import Protocol
+from openapi.openapi_client.model.spec_status import SpecStatus
+from openapi.openapi_client.model.volume_spec import VolumeSpec
+
+VOLUME_UUID = "5cd5378e-3f05-47f1-a830-a0f5873a1449"
+VOLUME_SIZE = 10485761
+NUM_VOLUME_REPLICAS = 1
+REST_SERVER = "http://localhost:8081/v0"
```

Comment:
It looks like this constant isn't used anywhere in the file.

Reply:
Yeah. This needs to be removed.